src/main/java/com/prgrms/amabnb/review/entity/Review.java

```diff
@@ -19,16 +22,24 @@ public class Review extends BaseEntity {
     @Id
     @GeneratedValue
     private Long id;
-
     private String content;
-
     private int score;
 
-    @Builder
-    public Review(Long id, String content, int score) {
+    @OneToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "reservation_id", unique = true)
+    private Reservation reservation;
+
+    public Review(Long id, String content, int score, Reservation reservation) {
         this.id = id;
         this.content = content;
         this.score = score;
+        this.reservation = reservation;
+    }
+
+    public Review(String content, int score, Reservation reservation) {
+        this.content = content;
+        this.score = score;
+        this.reservation = reservation;
     }
```

Comment:
```suggestion
    public Review(String content, int score, Reservation reservation) {
        this(null, content, score, reservation);
    }
```
주생성자 하나를 두고 부생성자를 만드는 식이 더 깔끔해보이네요!